pkg/service/service.go

```diff
@@ -12,6 +13,9 @@ import (
 	"github.com/redhat-developer/odo/pkg/util"
 )
 
+const provisionedAndBoundStatus = "ProvisionedAndBound"
+const provisionedAndLinkedStatus = "ProvisionedAndLinked"
```

Comment:
This looks good though

Reply:
These names are just what I picked, so I am of course open to alternatives :)